packages/slice-machine/src/features/builder/useSectionsExperiment.ts

```diff
@@ -0,0 +1,29 @@
+// import { useExperimentVariant } from "@/hooks/useExperimentVariant";
+
+export type useSectionsExperimentReturnType = {
```

Comment:
not sure we need to export this, but if so it should be 
```suggestion
export type UseSectionsExperimentReturnType = {
```
because it's a type

Reply:
Done ✅